presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -227,4 +227,10 @@ public static JsonParser createJsonParser(JsonFactory factory, Slice json)
         // so we pass an InputStreamReader instead.
         return factory.createParser(new InputStreamReader(json.getInput(), UTF_8));
     }
+
+    @Override
+    protected String applyLimit(String sql, long limit)
+    {
+        return sql + " LIMIT " + limit;
+    }
```

Comment:
What about Redshift?

Reply:
Good catch, I forgot about that. Updated